namespaces/live-1.cloud-platform.service.justice.gov.uk/poornima-dev/resources/rds.tf

```diff
@@ -0,0 +1,58 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cp_team_test_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.3"
```

Comment:
I think there was a release of the RDS module since you raised this PR! 
Please update this to use version 4.4

Reply:
Thanks David. I have updated the version number to the latest.